_posts/Blog Log Manangment Graylog

```diff
@@ -0,0 +1,61 @@
+
```

Comment:
---
short_name: maryroark
name: Mary Roark

github: graymary
linkedin: 'maryroark'
--
**Mary Roark** is a CISSP who is passionate about all things security

Reply:
Here is the author info